.github/workflows/automated_release.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - master
       - develop
+      - DPL-482-squix-printer-support
```

Comment:
Temp to allow me to deploy from branch. Will remove before merging!

Reply:
You can build from branch using github actions. Go to https://github.com/sanger/traction-ui > actions > Asset - on dispatch; build and publish > run workflow > use workflow from: <branch>, asset name: unique.